corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -89,3 +101,20 @@
   }
 
 }
+
+
+.question-container .gr .children {
+  .info.panel-default {
+    border: none;
+    background-color: transparent;
+    .box-shadow(none);
+
+    .panel-body {
+      padding-top: 6px;
+    }
+  }
+}
+
+.form-container .gr-header legend {
```

Comment:
Nitpick: can this block go inside the `.form-container` block at the top of this file?